string object_name
geometry_msgs/PoseStamped initial_pose
sensor_msgs/Image rgb_frame
sensor_msgs/Image depth_frame
string model_path
